REST/RESTListener+Handlers.cc

```diff
@@ -76,10 +76,10 @@ namespace litecore { namespace REST {
 #pragma mark - DATABASE HANDLERS:
 
     
-    void RESTListener::handleGetDatabase(RequestResponse &rq, C4Database *db) {
-        auto docCount = db->getDocumentCount();  // TODO: Collection-aware
-        auto lastSequence = db->getLastSequence();
-        C4UUID uuid = db->getPublicUUID();
+    void RESTListener::handleGetDatabase(RequestResponse &rq, C4Collection *coll) {
```

Comment:
Does it require to have scope and collection name in the result?

Reply:
I don't think so, since the user is requesting a specific collection via URL